docs/specs/localization.yml

```diff
@@ -2,92 +2,116 @@
 # [repository mapping] 
 commands:
   - restore --locale zh-cn
-  - build
-repo: https://github.com/docascode/docfx-test-dependencies#master
-restores:
-  git/github.com/docascode/docfx-test-dependencies.zh-cn/1dc2455be4a78b4fd29b5641e6fb8d687f37464a-master/dep.md: loc-DEP
```

Comment:
do you intend to remove all restores checks and the code in E2ETest?

Reply:
yes, I used the build output to check the restores

---
In reply to: [231407212](https://github.com/dotnet/docfx/pull/3585#discussion_r231407212) [](ancestors = 231407212)